### ISSUE TITLE
About License

### ISSUE THREAD

**Issue by Preysoft** [Project-OSRM/osrm-backend#6746]
Hello,
If I use the project's node module in my own project, I do not make any changes, I just use it in the code. Do I need to distribute my own project as open source due to the license?

**Comment by nilsnolde** [comment]
No, this is only GPL of the mainstream licenses which requires this. Here it's a very liberal BSD-2 clause license, so no, you do not have to distribute your project's source code.

Side note, as long as you don't distribute YOUR project to anyone external, there's no OSS license I'm aware of that requires you to distribute your code.

**Comment by Preysoft** [comment]
> No, this is only GPL of the mainstream licenses which requires this. Here it's a very liberal BSD-2 clause license, so no, you do not have to distribute your project's source code.
> 
> Side note, as long as you don't distribute YOUR project to anyone external, there's no OSS license I'm aware of that requires you to distribute your code.

Thank you for the answer.
People will use the project, but I will not share the project codes with people. Then, since I haven't made any changes to the project-osrm code, I don't have a license problem, right?
